### ISSUE TITLE
iOS App displays white screen when trying to use FlowListView from custom library

### ISSUE THREAD

**Issue by ScottMacDougall** [daniel-luberda/DLToolkit.Forms.Controls#41]
I'm looking to create a library that I can reference from a new Xamarin.Forms project. There will be a number of controls in the library, and FlowListView would be part of some of those controls; however on iOS 10.1 simulator (Haven't tried device or any other iOS versions) in both debug and release modes, when I try to run the app, all I get is a blank white screen. If I update the library to use a simple ListView instead of FlowListView, the app runs fine.

I have no problem using FlowListView directly in a project (not bundled in a custom library) and the library I created also works fine with FlowListView on an Android Simulator.

The library consists of 4 projects: Forms, iOS, and Android library projects, as well as a PCL for the Models and View Models.

**Comment by daniel-luberda** [comment]
Probably Xamarin's linker cuts it out. Call `FlowListView.Init()` somewhere in your project which uses your custom library.

**Comment by ScottMacDougall** [comment]
Sorry, I forgot to mention that I tried that.

I've tried calling Init from a static bootstrapper class in my library (from my project) as well as calling Init in just my project.

**Comment by daniel-luberda** [comment]
- Do you use nuget release for that or add dlls manually? 
- Do all projects in test app have that nuget added? 

Maybe you could share some code? It would be a lot harder to help you without it.

**Comment by ScottMacDougall** [comment]
Yes, nuget references in both the library and the calling project.

I'll whip up a sample project by the end of the weekend.

**Comment by daniel-luberda** [comment]
Sure, you could try to add dll file manually, maybe it would solve your issue. It's just a one dll only.

**Comment by Qwin** [comment]
I am having the same issue and I found out the issue is with iOS 10.1. If I use the iOS 9.3 emulator it works just fine.

**Comment by jamal-deen** [comment]
@Qwin did you find the solution? I am also facing the same problem in `iOS 10.1 `.

**Comment by daniel-luberda** [comment]
It may be related to Xamarin linking. Try to put `var ignore = new TagEntryView()` somewhere in code-behind.

**Comment by daniel-luberda** [comment]
Did you work it out?

**Comment by Qwin** [comment]
@daniel-luberda I worked it out however, I didn't get it to work with adding the assembly.  (just copied the files in my pcl project) I think it has to do with the PCL profile version + nuget adding my dll libraries with xamarin studio. Because when starting a new project it would work fine for my friend. I am using currently profile 111 + xamarin studio + ios 10.1. I tried also calling one of the controls from code behind so that I could see if it was a linker issue however that only gave the same error meaning issues importing the dll.
